doc/source/methods/PartialDependence.ipynb

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
<p>I'm think we should enforce some consistency between this page and the ALE one, in particular perhaps denoting the PD function with <code>PD</code> etc.? Keen to hear your thoughts. </p><p> </p><p>Monte Carol :))</p><p> </p><p>"we query the model on synthetic instance" -&gt; "we query the model on synthetic instances"</p><p>"Already that from the computation" -&gt; "Already from the computation"</p><p>"i.e. feature not correlated" -&gt; "i.e. features are not correlated"</p><p>"will create unrealistic synthetic instance" -&gt; "will create an unrealistic synthetic instance"</p><p>"that do not correlated" -&gt; "that are not correlated"</p><p>"hight" -&gt; "height"</p><p>"weight of child" -&gt; "weight of child"</p><p>"Such limitation can be addressed by ALE" -&gt; "This limitation can be addressed by the Accumulated Local Effects (ALE) method" - add link to ALE docs</p><p>"the PD still have" -&gt; "the PD still has"</p><p>"PD by setting all data instance" -&gt; "PD by setting all data instances"</p><p>"there exists some methods" -&gt; "there exist some methods"</p><p>"ALE to be extended to categorical feature, for" -&gt; "ALE to be extended to categorical features for"</p><p>"the&nbsp;<a href="https://christophm.github.io/interpretable-ml-book/ale.html" target="_blank">ALE chapter</a>&nbsp;from&nbsp;<a href="https://christophm.github.io/interpretable-ml-book/" target="_blank">Interpretable machine learning</a>&nbsp;book, C. Molnar(2020) as further reading." - need to rewrite, e.g. "from the Interpretable Machine Learning (C. Molnar, 2000) book" - can also look into using bibtex with <code>sphinx</code> (check <code>alibi-detect</code> overview of drift detection for how Ashley has done it)  </p><p> </p><p>"We will continue to discuss the simple case of linear regression." - I would replace this with a sentence to motivate this section, e.g. something like "We illustrate PD for the simple case of interpreting a linear regression model and show that it correctly results in showing a linear relationship between the features and the response."</p><p> </p><p>"assume the the features" -&gt; "assume the features"</p><p> </p><p>Comma at the end of the equation isn't needed since a new sentence is started. On that note probably good to include a finishing sentence that concludes what you set out to show (i.e. linear relationship).</p><p> </p><p>Link for Partial Dependence in the last sentence not working? Also worth pointing to the relevant ALE section too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Can you write the links to the ALE sections as I am not quite sure which ones you are referring to?</p>

Reply:
<p>Modified the links. Should work after the next release.</p><p> </p>